ingress/elb/elb.go

```diff
@@ -65,18 +75,33 @@ func (e *elb) Attach(frontend types.FrontendInput) (int, error) {
 		return 0, err
 	}
 
+	// Save these now so we can always know what we might have done
+	// up until this point we have only read data
+	e.elbs = clusterFrontEnds
```

Comment:
It may be a bit more robust to only store the ones successfully registered, so when it detaches it will avoid errors. But maybe not a big deal as long as we ignore the errors on detach.